Makefile

```diff
@@ -0,0 +1,120 @@
+
+test-infection-phpdbg-71: vendor
+	docker run -it --rm -v "$$PWD":/opt/infection -w /opt/infection php:7.1-cli phpdbg -qrr bin/infection --threads=4
+
+test-infection-phpdbg-72: vendor
+	docker run -it --rm -v "$$PWD":/opt/infection -w /opt/infection php:7.2-cli phpdbg -qrr bin/infection --threads=4
+
+
+.PHONY: test-e2e-phpdbg test-e2e-phpdbg-70 test-e2e-phpdbg-71 test-e2e-phpdbg-72
+#e2e tests with phpdbg
+test-e2e-phpdbg: test-e2e-phpdbg-70 test-e2e-phpdbg-71 test-e2e-phpdbg-72
+
+test-e2e-phpdbg-70: vendor
+	docker run -it --rm -e PHPDBG=1 -v "$$PWD":/opt/infection -w /opt/infection php:7.0-cli ./tests/e2e_tests
+
+test-e2e-phpdbg-71: vendor
+	docker run -it --rm -e PHPDBG=1 -v "$$PWD":/opt/infection -w /opt/infection php:7.1-cli ./tests/e2e_tests
+
+test-e2e-phpdbg-72: vendor
+	docker run -it --rm -e PHPDBG=1 -v "$$PWD":/opt/infection -w /opt/infection php:7.2-cli ./tests/e2e_tests
+
+
+.PHONY: test-infection-xdebug test-infection-xdebug-70 test-infection-xdebug-71 test-infection-xdebug-72
+#infection with xdebug
+test-infection-xdebug: test-infection-xdebug-70 test-infection-xdebug-71 test-infection-xdebug-72
+
+test-infection-xdebug-70: build-xdebug-70
+	docker run -it --rm -v "$$PWD":/opt/infection -w /opt/infection infection_php70 php bin/infection --threads=4
+
+test-infection-xdebug-71: build-xdebug-71
+	docker run -it --rm -v "$$PWD":/opt/infection -w /opt/infection infection_php71 php bin/infection --threads=4
+
+test-infection-xdebug-72: build-xdebug-72
+	docker run -it --rm -v "$$PWD":/opt/infection -w /opt/infection infection_php72 php bin/infection --threads=4
+
+
+.PHONY: test-e2e-xdebug test-e2e-xdebug-70 test-e2e-xdebug-71 test-e2e-xdebug-72
+#e2e tests with xdebug
+test-e2e-xdebug: test-e2e-xdebug-70 test-e2e-xdebug-71 test-e2e-xdebug-72
+
+test-e2e-xdebug-70: build-xdebug-70
+	docker run -it --rm -v "$$PWD":/opt/infection -w /opt/infection infection_php70 ./tests/e2e_tests
+
+test-e2e-xdebug-71: build-xdebug-71
+	docker run -it --rm -v "$$PWD":/opt/infection -w /opt/infection infection_php71 ./tests/e2e_tests
+
+test-e2e-xdebug-72: build-xdebug-72
+	docker run -it --rm -v "$$PWD":/opt/infection -w /opt/infection infection_php72 ./tests/e2e_tests
+
+
+.PHONY: build-xdebug-70 build-xdebug-71 build-xdebug-72
+#Building images with xdebug
+build-xdebug-70: vendor
+	docker build -t infection_php70 -f "$$PWD/devTools/Dockerfile-php70-xdebug" .
+
+build-xdebug-71: vendor
+	docker build -t infection_php71 -f "$$PWD/devTools/Dockerfile-php71-xdebug" .
+
+build-xdebug-72: vendor
+	docker build -t infection_php72 -f "$$PWD/devTools/Dockerfile-php72-xdebug" .
+
+#style checks/ static analysis
+.PHONY: analyze cs-fix cs-check phpstan validate update
+analyze: cs-check phpstan validate
+
+cs-fix: build/cache php-cs-fixer-v2.phar
+	./php-cs-fixer-v2.phar fix -v --cache-file=build/cache/.php_cs.cache
+
+cs-check: build/cache php-cs-fixer-v2.phar
```

Comment:
please update this file with the new commands 

https://github.com/infection/infection/blob/master/.github/CONTRIBUTING.md#how-to-contribute